CONTRIBUTING.md

```diff
@@ -0,0 +1,148 @@
+# Contributing to opentelemetry-ebpf-profiler
+
+The Go special interest group (SIG) meets regularly. See the
+OpenTelemetry
+[community](https://github.com/open-telemetry/community)
+repo for information on this and other SIGs.
+
+See the [public meeting
+notes](https://docs.google.com/document/d/19UqPPPlGE83N37MhS93uRlxsP1_wGxQ33Qv6CDHaEp0/edit#heading=h.4rdgawyis2hd)
+for a summary description of past meetings.
+
+## Development
+
+You can view and edit the source code by cloning this repository:
+
+```sh
+git clone https://github.com/open-telemetry/opentelemetry-ebpf-profiler.git
+```
+
+Run `make test` to run the tests instead of `go test`.
+
+Additionally, there is a `codespell` target that checks for common
+typos in the code. It is not run by default, but you can run it
+manually with `make codespell`.
```

Comment:
There are multiple Make targets. Might be a personal view, but `make codespell` is not important to get started. To start with a project, targets on how to build and how to test are the most important ones, I think.
Otherwise, the question arises, why `codespell` should be mentioned here but not `lint`.
```suggestion
```